common/src/main/resources/tachyon-default.properties

```diff
@@ -92,6 +92,10 @@ tachyon.worker.tieredstore.level.max=1
 tachyon.worker.tieredstore.level0.alias=MEM
 tachyon.worker.tieredstore.level0.dirs.quota=${tachyon.worker.memory.size}
 tachyon.worker.tieredstore.level0.dirs.path=/mnt/ramdisk
+tachyon.worker.tieredstore.evict.async.enable=false
```

Comment:
I think its better to have a default reserved ratio value here in tachyon-default.properties